ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -317,8 +317,8 @@ tests:
     test:
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi
-- as: aws-ipi-f9-longduration-hive-sd-rosa
-  cron: 5 19 7,16,25 * *
+- as: aws-ipi-longduration-hive-sd-rosa-f7
```

Comment:
The CI Jobs are ordered in alphabetical order strictly. If you change the job name, need make it in the right place.